test/integration/integration_admin_test.cc

```diff
@@ -34,27 +34,90 @@ INSTANTIATE_TEST_SUITE_P(Protocols, IntegrationAdminTest,
                              {Http::CodecType::HTTP1})),
                          HttpProtocolIntegrationTest::protocolTestParamsToString);
 
+namespace {
+
+// NOLINTNEXTLINE(readability-identifier-naming)
+std::string ContentType(const BufferingStreamDecoderPtr& response) {
```

Comment:
rename to `contentType` and drop the lint override. Also it could return a string_view rather than copying into a string.

I realize you are just moving the pre-existing code but it's being looked at through a new lens of code standards and lint tools :)

thanks :)